Stream-3.0/src/main/java/com/example/Stream30/Controller/EmployeeController.java

```diff
@@ -0,0 +1,45 @@
+package com.example.Stream30.Controller;
+
+import com.example.Stream30.Model.Employee;
+import com.example.Stream30.Service.EmployeeService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping(path = "/departments")
+public class EmployeeController {
+    private final EmployeeService employeeService;
+
+    public EmployeeController(EmployeeService employeeService) {
+        this.employeeService = employeeService;
+    }
+
+    @GetMapping(path = "/min-salary")
+    public Employee getMinSalary(int departmentId){
+        return employeeService.getMinSalary(departmentId);
+
+    }
+    @GetMapping (path = "/max-salary")
+
+    public Employee getMaxSalary(int departmentId){
+        return employeeService.getMaxSalary(departmentId);
+
+    }
+    @GetMapping (path = "/all")
```

Comment:
здесь надо указать параметр, чтобы разрешить конфликт одинаковых путей /all
```suggestion
    @GetMapping (path = "/all", params = "departmentId")
```